src/components/Articles/Articles.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-card flat id="articles">
+    <confirm-dialog ref="confirm" attach="#app" />
+
+    <div class="define-article-header">
+      <v-icon color="app-dk-blue">mdi-handshake</v-icon>
+      <label class="define-article-title">Articles</label>
+    </div>
+
+    <div class="section-container">
+      <company-provisions
+        class="sub-section"
+        :provisionsRemoved="getProvisionsRemoved"
+        @companyProvisionsChanged="updateProvisionsRemoved($event)"
+        @haveChanges="emitHaveChanges($event)"/>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Mixins, Watch } from 'vue-property-decorator'
+import { CommonMixin } from '@/mixins'
+import { ConfirmDialog } from '@/components/dialogs'
+import CompanyProvisions from './CompanyProvisions.vue'
+import { Action, Getter } from 'vuex-class'
+import { ActionBindingIF } from '@/interfaces'
+
+@Component({
+  components: {
+    ConfirmDialog,
+    CompanyProvisions
+  }
+})
+export default class Articles extends Mixins(CommonMixin) {
+  // whether components have changes
+  private companyProvisionsChanges = false
+
+  @Getter getProvisionsRemoved!: boolean
+
+  // Setters
+  @Action setProvisionsRemoved!: ActionBindingIF
+
+  private updateProvisionsRemoved (provisionsRemoved: boolean): void {
```

Comment:
If you touch this file again, consider removing this method, since you can call `setProvisionsRemoved` directly in the template.
If not, it's fine.

Reply:
Done.